src/App.jsx

```diff
@@ -0,0 +1,46 @@
+import './App.css';
+import { useState } from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Login from './Component-Routes/Login/Login'
+import MakeNote from './Component-Routes/MakeNote/MakeNote';
+import Feed from '../src/Component-Routes/Feed/Feed';
+import Register from './Component-Routes/Register';
+import Page404 from './Component-Routes/Page404/Page404';
```

Comment:
```suggestion
import './App.css';
import { useState } from 'react';

import { BrowserRouter, Routes, Route } from 'react-router-dom';

import Login from './Component-Routes/Login/Login'
import AddNote from './Component-Routes/MakeNote/AddNote';
import Feed from '../src/Component-Routes/Feed/Feed';
import Register from './Component-Routes/Register';
import Page404 from './Component-Routes/Page404/Page404';
```
Solo un par de entres para distinguir fácilmente 
Sobre Makenote  podrías renombrarlo por AddNote